src/parser/jobs/blu/modules/AlwaysBeCasting.tsx

```diff
@@ -143,6 +156,23 @@ export class AlwaysBeCasting extends CoreAlwaysBeCasting {
 	override onComplete() {
 		super.onComplete()
 
+		this.diamondBackHistory.closeCurrent(this.parser.pull.timestamp + this.parser.pull.duration)
+		const diamondBackTimeMs = this.diamondBackHistory.entries.reduce((acc, e) => {
+			return acc + ((e.end ?? e.start) - e.start)
+		}, 0)
+		const diamondBackSeconds = Math.ceil(diamondBackTimeMs / 1000)
+		this.suggestions.add(new TieredSuggestion({
+			icon: this.data.actions.DIAMONDBACK.icon,
+			content: <Trans id="blu.diamondback.time.content">
+				The ABC report will count time spent under <DataLink action="DIAMONDBACK" /> as GCD uptime, but you should still aim to minimize this, since it means dropping damaging GCDs.
+			</Trans>,
+			why: <Trans id="blu.diamondback.time.why">
+				<Plural value={diamondBackSeconds ?? 0} one="# second" other="# seconds" /> spent in Diamondback.
+			</Trans>,
+			tiers: {1: SEVERITY.MINOR},
+			value: diamondBackSeconds,
```

Comment:
I feel like this would make more sense as just a count of how many times Diamondback was used. You're always going to spend ten seconds unable to act per cast (unless you manage to die anyways I guess), so I'm not sure what value we're gaining by displaying it as time here.

Reply:
There are boss actions that will knock you out of Diamondback early, so assuming 10s for every cast is not always true (e.g. O4S phase 2).

Reply:
Aye, A8S also has a similar "you get knocked out of Diamondback".  It's also fine to spend time in Diamondback during some downtime mechanics, though this patch isn't handling that

Reply:
Aha, that wasn't something I was aware of. Fair enough then, we'll leave it as a time!